crates/core/c8y_smartrest/src/error.rs

```diff
@@ -43,6 +43,9 @@ pub enum SmartRestDeserializerError {
 
     #[error("Empty request")]
     EmptyRequest,
+
+    #[error("The received SmartREST message payload {payload} is an invalid payload")]
+    InvalidMessagePayload { payload: String },
```

Comment:
This can be removed now as `get_smartrest_template_id()` always returns `String`.

Reply:
fixed